runtime/v1/linux/runtime.go

```diff
@@ -338,12 +339,7 @@ func (r *Runtime) loadTasks(ctx context.Context, ns string) ([]*Task, error) {
 		ctx = namespaces.WithNamespace(ctx, ns)
 		pid, _ := runc.ReadPidFile(filepath.Join(bundle.path, proc.InitPidFile))
 		s, err := bundle.NewShimClient(ctx, ns, ShimConnect(r.config, func() {
-			_, err := r.tasks.Get(ctx, id)
```

Comment:
I also aware in loadTasks `t.tasks`  is empty, so we do not need check task available here ? @Random-Liu

Reply:
This is called later on close, so we still need to check this.

The error handling below doesn't need to check, because the task is not in list yet.

Reply:
Yes, you are right, I miss the point